### PR TITLE
Homepage redesign

### PR DESCRIPTION
**Pull request by alesgenova**
screenshot_2018-09-11 open chemistry

### CODE REVIEW

**Comment by cjh1** [comment]
Lets not commit the CJSON for the molecule. We can just pull one from Girder.

**Comment by cjh1** [comment]
@alesgenova This is looking really good.

**Comment by cryos** [comment]
I think you should edit the history to remove the JSON for the molecule from the first commit, or squash the first and second commit together.

File: src/components/home/index.js [hunk 1]
```diff
@@ -123,16 +123,24 @@ class Home extends Component {
             <Grid container spacing={24} alignItems="center">
               <Grid item xs={12} md={7}>
                 <Typography color="inherit" gutterBottom variant="display1">
-                  Open Chemistry
+                  Open Chemistry Data Plaform
```
Comment (cjh1) [inline src/components/home/index.js]: typo Plaform => Platform

Comment (cryos) [reply src/components/home/index.js]: Good catch, thanks Chris

Comment (cjh1) [reply src/components/home/index.js]: Not bad for a dsylieixic :smile:

**Pull request merged**